packages/manager/apps/telecom/src/app/telecom/pack/move/pack-move.controller.js

```diff
@@ -268,33 +266,25 @@ export default class PackMoveCtrl {
    */
   isSlammingLine() {
     this.slammingCheck = true;
-    return this.OvhApiPackXdsl.v7()
-      .access()
-      .execute({
-        packName: this.packName,
+    return this.OvhApiPackXdsl.Access()
+      .v6()
+      .getServices({
+        packId: this.packName,
       })
-      .$promise.then((ids) =>
-        this.$q
-          .all(
-            map(
-              chunk(ids, 200),
-              (chunkIds) =>
-                this.OvhApiXdsl.v7()
-                  .query()
-                  .batch('serviceName', [''].concat(chunkIds), ',')
-                  .expand()
-                  .execute().$promise,
-            ),
-          )
-          .then((chunkResult) => flatten(chunkResult))
-          .then((result) => flatten(result)),
-      )
-      .then((xdslLines) => {
-        const slammingLines = filter(
-          xdslLines,
-          (xdslLine) => xdslLine.value.status === LINE_STATUS.slamming,
-        );
-        this.hasSlamming = !!slammingLines.length;
+      .$promise.then((ids) => {
+        const request = this.iceberg('/xdsl')
+          .query()
+          .expand('CachedObjectList-Pages')
+          .addFilter('status', 'eq', LINE_STATUS.slamming);
+        if (ids.length > 1) {
+          request.addFilter('accessName', 'in', ids);
+        } else {
+          request.addFilter('accessName', 'eq', ids);
```

Comment:
isn't ```ids```always an array ? in that case ```in``` operator should always work

Reply:
Will try to remember. Was done on december 😅

Reply:
```400 Bad Request : {"message":"in/nin operator must contains at least 2 values separated by comma"}``` :/